docs-src/0.4/en/cookbook/optimizing.md

```diff
@@ -0,0 +1,131 @@
+# Optimizing
+
+*Note: This is written primarily for the web, but the main optimizations will work on other platforms too.*
+
+You might have noticed that Dioxus binaries are pretty big.
+The WASM binary of a [TodoMVC app](https://github.com/tigerros/dioxus-todo-app) weighs in at 2.36mb!
+Don't worry; we can get it down to a much more manageable 234kb.
+This will get obviously lower over time.
+For example, the new [signals](https://github.com/DioxusLabs/dioxus/tree/master/packages/signals) will reduce the binary size of a hello world app to less than 100kb (with unstable features).
```

Comment:
I don't think signals will make the binary size smaller. https://github.com/DioxusLabs/dioxus/pull/1402 will

Reply:
Yeah, I got it mixed up. Let me fix it right away.